src/test/java/com/hillel/items_exchange/end2end/AdvertisementFlowTest.java

```diff
@@ -203,4 +296,28 @@ void setDefaultImage_success() throws Exception {
                 .andDo(print())
                 .andExpect(status().isOk());
     }
+
+    @Test
+    @WithMockUser(username = "deletedUser")
+    @Transactional
+    @DataSet({"database_init.yml", "user/deleted_user_init.yml"})
+    void setDefaultImage_WhenUserHasStatusDeleted_ShouldReturn403WithSpecificMessage() throws Exception {
+        MvcResult mvcResult = mockMvc.perform(post("/adv/default-image/{advertisementId}/{imageId}",
+                validId6, validId))
+                .andDo(print())
+                .andExpect(status().isForbidden())
+                .andReturn();
+
+        User deletedUser = userService.findByUsernameOrEmail("deletedUser@gmail.com").orElseThrow();
+        String contentAsString = mvcResult.getResponse().getContentAsString();
+
+        assertTrue(contentAsString.contains(getIllegalOperationMessage(deletedUser)));
```

Comment:
@svyatoslav145 I see such duplication more than 8 times. Why do we need to test the same in every test?

Reply:
the same thing